### PR TITLE
Login user

### PR DESCRIPTION
**Pull request by jvaneyken**
## Description
Remove AuthForm Component, move register form into RegisterPage component and login form into LoginPage component, implement login functionality 

## Related Issue
Closes https://github.com/dyazdani/zoop/issues/53
Closes https://github.com/dyazdani/zoop/issues/54

### On register, token is generated, state is updated with the token and it is logged to the console

Screenshot 2023-11-12 at 1 59 53 PM

### On page refresh, token is no longer in state and is not logged to the console

Screenshot 2023-11-12 at 2 00 18 PM

### On login, token is once again generated and logged to the console

Screenshot 2023-11-12 at 2 02 02 PM

### CODE REVIEW

File: client/components/LoginPage.tsx [hunk 1]
```diff
@@ -1,9 +1,47 @@
...
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+
+  const handleSubmit = (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    login({ email, password });
+  };
+
+  return (
+    <>
```
Comment (paloobi) [inline client/components/LoginPage.tsx:16]: No need for this wrapping fragment if you're returning a single form element - just return the form

File: client/components/RegisterPage.tsx [hunk 2]
```diff
@@ -1,13 +1,67 @@
...
+  const [password, setPassword] = useState("");
+  const [confirmPassword, setConfirmPassword] = useState("");
+
+  const handleSubmit = (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    register({ email, username, password });
+  };
+
+  return (
+    <>
```
Comment (paloobi) [inline client/components/RegisterPage.tsx:18]: Same: delete unneeded wrapping fragment

**Review by paloobi — APPROVED** [review]
Recommend removing unneeded fragments wrapping fragments, but otherwise lgtm

I'm assuming you'll be adding mui in a separate PR?

**Comment by jvaneyken** [comment]
> Recommend removing unneeded fragments wrapping fragments, but otherwise lgtm
> 
> I'm assuming you'll be adding mui in a separate PR?

Yes, I believe that is the plan. Hoping to do that ASAP.

**Pull request merged**